### PR TITLE
feat: add support for range validation

### PR DESCRIPTION
**Pull request by razvanz**

### CODE REVIEW

File: src/js/date-field.js [hunk 1]
```diff
@@ -5,6 +5,21 @@ var parseDate = require('i18n-parse-date'),
 module.exports = require('ember-text-field').extend({
     autocomplete: 'off',
 
+    min: null,
+    max: null,
+
+    init: function() {
```
Comment (fiolkaf) [inline src/js/date-field.js]: init is already defined: https://github.com/billysbilling/ember-date-field/pull/3/files#diff-758f4c4248fb7fd991a7c5e3bf077f42R33

Do we really want to support dates s numbers?

**Review by fiolkaf — CHANGES_REQUESTED** [review]

**Comment by fandridis** [comment]
Should we change it to look to `develop` branch first?

**Comment by razvanz** [comment]
There's no develop branch in this repo

**Review by fandridis — APPROVED** [review]
Looks like a good addition to have. 👍

**Pull request merged**